R/utilities-molecule.R

```diff
@@ -107,7 +107,7 @@ setMoleculeInitialValues <- function(molecules, values) {
 #' setMoleculeScaleDivisors(molecules, c(0.002, 0.003))
 #' @export
 setMoleculeScaleDivisors <- function(molecules, values) {
-  molecules <- c(molecules)
+  molecules <- enforceIsList(molecules)

```

Comment:
Isn't it equivalent? What is the benefits of this function?

Reply:
In this case it is. I just hate it that `c()` might return either a vector (e.g. `c(1, 2 , 3)`) or a list (in case the values are not atomic).

Reply:
oh I see. Ok. should we maybe just rename the method to something less violent? asList or toList?